app.js

```diff
@@ -401,11 +401,16 @@
 
       getComments: function(macro) {
         var actions = _.filter(macro.actions, function(action) {
-          return action.value && action.field == "comment_value";
+          return action.value && (["comment_value", "comment_value_html"].includes(action.field));
         });
         var comments = _.map(actions, function(action) {
-          return action.value[1].toLowerCase();
-        });
+          //Rich content macros' value is a string instead of an array. A string full of html.
+          if (typeof(action.value) == "string") {
+            return this.stripHTMLTags(action.value.toLowerCase());
+          } else {
+            return action.value[1].toLowerCase();
+          }
+        }.bind(this));
```

Comment:
`{this.("I did this with this this because binding this this allows this function to have access to this.stripHTMLTags.")}.bind(this)`